types/leaflet/index.d.ts

```diff
@@ -1128,9 +1137,9 @@ export interface PanOptions {
     noMoveStart?: boolean;
 }
 
-/* tslint:disable:no-empty-interface */ // This is not empty, it extends two interfaces into one...
+// This is not empty, it extends two interfaces into one...
+// tslint:disable-next-line
```

Comment:
In older tslint versions I know the problem with `no-empty-interface`, but I think you are now able to add two interfaces into one without having trouble with the linter. So keep the line above changed and remove this `disable-next-line` like mentioned above...